docs/mkdocs/reference/lineapy/api/api.md

```diff
@@ -0,0 +1 @@
+::: lineapy.api.api
```

Comment:
We'll need to convert these too. Adding a ticket LIN-743 to track this.

Reply:
Electing to skip these, the internals documentation was left out in the previous PR https://github.com/LineaLabs/lineapy/pull/856 probably because the contents will simply clutter and confuse public facing readers. LIN-743 has been updated to convert these docs to internal docs.